crates/transaction-emitter-lib/src/transaction_generator/publishing/module_simple.rs

```diff
@@ -100,129 +100,160 @@ pub fn scramble(module: &mut CompiledModule, fn_count: usize, rng: &mut StdRng)
 //
 // List of entry points to expose
 //
-enum EntryPoints {
+#[derive(Debug, Copy, Clone)]
+pub enum EntryPoints {
     // 0 args
-    Nop = 0,
-    Step = 1,
-    GetCounter = 2,
-    ResetData = 3,
-    Double = 4,
-    Half = 5,
+    Nop,
```

Comment:
nit: I know this is unrelated to your change,  but these enums names are not self-explanatory and could use some comment to explain their behavior.